src/emc/usr_intf/stepconf/axisa.glade

```diff
@@ -1,5 +1,9 @@
+<<<<<<< HEAD
```

Comment:
This merge commit adds [conflict markers.](http://stackoverflow.com/questions/7901864/git-conflict-markers), which means the resulting glade files will not work.  This happened because of some other modifications to the glade files in our master branch since you submitted this pull request ("stepconf: dynamically show how step scale is calculated" by @c-morley).  I can help you fix this problem.